frontend/server/plugins/sentry.ts

```diff
@@ -0,0 +1,35 @@
+import { useRuntimeConfig } from "#imports"
+
+import { defineNitroPlugin } from "nitropack/runtime"
+import * as Sentry from "@sentry/node"
+
+import { logger } from "~~/server/utils/logger"
+
+export default defineNitroPlugin((nitroApp) => {
+  const {
+    public: { sentry },
+  } = useRuntimeConfig()
+
+  if (import.meta.dev) {
+    // Should not need this, but without it Dev builds fail with "The requested module 'vue' does not provide an export named 'computed'"
+    // @see https://github.com/getsentry/sentry-javascript/issues/12490
+    // @ts-expect-error - globalThis is not defined here, and this is only used in dev mode.
+    globalThis._sentryEsmLoaderHookRegistered = true
+  }
```

Comment:
Good news, this appears to be unnecessary with @sentry/node "^8.20.0"!

https://github.com/getsentry/sentry-javascript/issues/12490#issuecomment-2214961942

I tested locally with `j f` and it did not show any error. Is that what the problem was, or did it fail on dev _builds_ specifically, rather than the dev local server?

Reply:
Problem is indeed fixed, thank you!